docs/notebooks/ordinal_regression.ipynb

```diff
@@ -0,0 +1,1844 @@
+{
```

Comment:
<p>In the first model, when I omit the intercept <code>response ~ 0 </code> the warning is no longer raised. I would need to explicitly include a model with the intercept to explain the behavior. Instead I explain this behavior in the paragraph before fitting the "threshold-only" model.</p>

Reply:
<p>In this model, if I omit the intercept I get <code>IndexError: list index out of range </code> originating from <code>formulae</code>.</p><p> </p><pre class="ql-syntax" spellcheck="false">File ~/Documents/repos/bambi/bambi/models.py:157, in Model.__init__(self, formula, data, family, priors, link, categorical, potentials, dropna, auto_scale, noncentered, center_predictors, extra_namespace)
    153     self.formula = check_ordinal_formula(self.formula)
    154     # Notice the intercept is added so formulae constrains categorical predictors, avoiding
    155     # linear dependencies with the cutpoints.
    156     # Then the intercept is removed from the design matrix because of the cutpoints.
--&gt; 157     design = fm.design_matrices(
    158         self.formula.main + " + 1", self.data, na_action, 1, additional_namespace
    159     )
    160     design = remove_common_intercept(design)
    161 else:

File ~/miniforge3/envs/bambinos/lib/python3.11/site-packages/formulae/matrices.py:556, in design_matrices(formula, data, na_action, env, extra_namespace)
    553     else:
    554         raise ValueError(f"'data' contains {incomplete_rows_n} incomplete rows.")
--&gt; 556 design = DesignMatrices(description, data, env)
...
-&gt; 1255         encoding = encodings[term.name][0]
   1256     else:
   1257         encoding = False

IndexError: list index out of range  
</pre>

Reply:
<p>I <em>think</em> it's because you need to update formulae. I still need to to the release, but in the meantime </p><p> </p><pre class="ql-syntax" spellcheck="false">pip install git+https://github.com/bambinos/formulae.git --force-reinstall
</pre><p> </p><p>Just make sure you don't get numpy 1.26.0 because it doesn't work with pytensor now.</p>

Reply:
This resolved the error. Thanks!